src/lib/integration_test_local_engine/mina_automation.ml

```diff
@@ -0,0 +1,462 @@
+open Core
```

Comment:
File name nit: It's a little confusing to call this module `mina_automation` given that it doesn't interact with our `automation/` codebase. Recommend renaming to `Docker_network` or `Mina_docker` or something like that.